spyt-package/src/main/python/spyt/conf.py

```diff
@@ -26,6 +18,71 @@
 logger = logging.getLogger(__name__)
 
 
+class SpytDistributions:
```

Comment:
I'd rename this class and the `get_spyt_distributions`  method to `SpytDistribution` because plural form confuses. It actually points to a different root, but it's not multiple distributions.

Reply:
But it is really not singular, but plural, because this yt_root contains multiple SPYT and Spark distributions and to choose one you have to provide specific version to it.

Reply:
Why do you need to support multiple spyt and spark distributions for a single cluster or job? I think there must be specific versions for each YT operation.

Reply:
The catalog `//home/spark` is organized in a such way that there are multiple versions in it. If I am giving a way to change this path to something else, I am giving way to customize multiple SPYT distributions, not a single one, even though only one will be used.